ipyleaflet/leaflet.py

```diff
@@ -1261,10 +1261,16 @@ class GeoData(GeoJSON):
     def __init__(self, **kwargs):
         super(GeoData, self).__init__(**kwargs)
         self.data = self._get_data()
+        self.data = GeoJSON._get_data(self)
```

Comment:
Does it make sense to call `self.data = GeoJSON._get_data(self)` just after `self.data = self._get_data()` ? You overwrites its value making `self._get_data()` useless

Reply:
`self._get_data()` just converting geodata frame to json and actual logic of `style` and `style_callback` is already implemented in `GeoJSON._get_data` hence reusing that.